multiaccounts/settings/columns.go

```diff
@@ -324,6 +324,11 @@ var (
 		dBColumnName:   "remote_push_notifications_enabled",
 		valueHandler:   BoolHandler,
 	}
+	NewsPushNotificationsEnabled = SettingField{
+		reactFieldName: "news-push-notifications-enabled?",
```

Comment:
Why do some fields end with `?` and some not? Do they need to have `?`?

Reply:
I think this is a convention for how we name boolean fields in ClojureScript. We may not be consistent with all the boolean fields for these columns, but this one happens to match the other push-notification settings.